docs/resources/ces_alarmrule.md

```diff
@@ -148,7 +151,22 @@ The `condition` block supports:
 
 * `count` - (Required, Int) Specifies the number of consecutive occurrence times. The value ranges from 1 to 5.
 
-* `unit` - (Optional, String) Specifies the data unit.
+* `unit` - (Optional, String, ForceNew) Specifies the data unit. Changing this creates a new resource.
```

Comment:
Shall we give some examples for `unit` according to the type of rule?

Reply:
sure, I add a link https://support.huaweicloud.com/intl/en-us/api-ces/ces_03_0059.html